website/translated_docs/fr/concepts/function-components.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Composants fonctionnels
+sidebar_label: introduction
+description: Introduction aux fonctions composants
+---
+
+::: avertissement Nous travaillons toujours sur les composants de fonction et les Hooks. Ils ne sont pas encore tout à fait prêts à être utilisés. Si vous souhaitez aider, jetez un œil au [tableau de projet](https://github.com/yewstack/yew/projects/3) pour une liste des choses qui doivent encore être faites. :::
```

Comment:
This is a bit broken unfortunately... It seems like GitLocalize doesn't handle these blocks properly.
"warning" shouldn't be translated at all
```suggestion
:::warning
Nous travaillons toujours sur les composants de fonction et les Hooks. Ils ne sont pas encore tout à fait prêts à être utilisés.
Si vous souhaitez aider, jetez un œil au [tableau de projet](https://github.com/yewstack/yew/projects/3) pour une liste des choses qui doivent encore être faites.
:::
```

Reply:
Yep, could you please close this PR? I was just testing it out sorry

Reply:
Yep, I'll fix that